### DISCUSSION TITLE
Frontend Prompts

### DISCUSSION THREAD

**Discussion by lDrakohh** [ispp-2324-ocial/KB#2] [Prompts IAS]
Insert here prompts use to generate frontend content.

**Comment by AitorRD** [comment]
Se ha utilizado este prompt como base para el artículo del primer post del blog de la landing-page: https://chat.openai.com/share/1c765903-6eda-4745-82aa-1055e7509248

**Comment by ferferga** [comment]
Se ha utilizado este prompt para saber la diferencia entre tipografías variables y estáticas:
https://sl.bing.net/fTqYgQIGu28
